src/basic/env-util.c

```diff
@@ -183,36 +183,34 @@ static int env_append(char **r, char ***k, char **a) {
         return 0;
 }
 
-char **strv_env_merge(size_t n_lists, ...) {
+char** _strv_env_merge(char **first, ...) {
```

Comment:
we usually call these xyz_internal() or so.

Reply:
We use conventions… "_internal" is the more verbose one, and "_" is the python-inspired alternative. I prefer the latter because it conveys the same information in 1/9th of the characters.